userspace/libsinsp/sinsp.h

```diff
@@ -253,6 +253,16 @@ class SINSP_PUBLIC sinsp
 	*/
 	void open(string filename);
 
+	/*!
+	  \brief Start an event capture from a file descriptor.
+
+	  \param filename the trace file name.
```

Comment:
Oh good catch. I won't make that change on this PR, though. I'll do it on a PR for dev.